Baraka-Mugisha/MyDiary
# #169564121 Changes Tests directories

#### What does this PR do?
`Changes tests directories`

#### Description of Task to be completed?
`exported v1 tests`
`imported v1 tests in v2`

#### How should this be manually tested?
`do npm install`
`check the folder templates`
`test the routes of version one with POSTMAN`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#169564121

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
